_pages/production.md

```diff
@@ -46,28 +46,20 @@ Make sure you have the following items ready before you start the deployment pro
 
 Depending on your agency’s integration additional items may be needed:
 
-- If this is a SAML integration (not OpenID Connect), then please ensure that:
+- **If this is a SAML integration (not OpenID Connect), then please ensure that:**
 
-  -   **Assertion Consumer Logout Service URL** is defined.
+  -   Assertion Consumer Logout Service URL is defined.
 
-  -   **SAML Assertion Encryption** is enabled.
+  -   SAML Assertion Encryption is enabled.
 
         -   If you are using a service which does not support SAML encryption, then please submit a technical support ticket through the [Partner Support Help Desk](https://zendesk.login.gov) for further guidance.
 
-- If this is an integration requesting identity proofed attributes, you must include a Failure to Proof URL. Users will be redirected to this URL if they fail to complete the identity verification process. This page should communicate your agency and/or departments alternate methods of accessing your application.
-
-If you have questions after reviewing this page, submit a technical support ticket through the [Partner Support Help Desk](https://zendesk.login.gov). You will need a [Login.gov production account](https://secure.login.gov) to submit technical support tickets. Your Login.gov production account and Login.gov sandbox (test environment) accounts are separate.
-
--   If this is a SAML integration (Not OpenID Connect), then please ensure that:
-
-    -   Assertion Consumer Logout Service URL is defined.
+- **If this is an integration requesting identity proofed attributes, you must include a Failure to Proof URL.** Users will be redirected to this URL if they fail to complete the identity verification process. This page should communicate your agency and/or departments alternate methods of accessing your application.
 
-    -   SAML Assertion Encryption is enabled.
+If you have questions after reviewing this page, submit a technical support ticket through the [Partner Support Help Desk]({{ site.baseurl}}/support/#contacting-partner-support). You will need a [Login.gov production account](https://secure.login.gov) to submit technical support tickets. Your Login.gov production account and Login.gov sandbox (test environment) accounts are separate.
```

Comment:
flagging Sandbox here - but this is a case where I am not sure replacing sandbox with dashboard is entirely accurate.

Reply:
sandbox is correct here @karenstrother ! (it is differentiating between login.gov accounts.) arguably we should call it the integration environment rather than test environment? (but we're not very consistent about that syntax.)

Reply:
yeah I agree with Davida's comment below that in this case we are describing the accounts so I don't think we want to change the language here